### ISSUE TITLE
Parity stopped syncing the block #2714664

### ISSUE THREAD

**Issue by chakhalyan1** [paritytech/parity-ethereum#10555]
_Before filing a new issue, please **provide the following information**._

- **Parity Ethereum version**: 2.5.0
- **Operating system**: Linux
- **Installation**: built from source
- **Fully synchronized**: no 
- **Network**: ethereum
- **Restarted**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._
Hi All.
 I have remove chains/network/nodes.json file before starting parity  and started with **./parity --pruning archive --min-peers 6 -l sync=debug**
do not continue sync process after this block #2714664
and log is following.
```
2019-04-01 10:37:59  IO Worker #1 INFO import     2/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 15 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-01 10:38:29  IO Worker #3 INFO import     2/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 14 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-01 10:38:59  IO Worker #0 INFO import     3/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 14 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-01 10:39:29  IO Worker #2 INFO import     3/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 14 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-01 10:39:59  IO Worker #0 INFO import     3/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 15 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-01 10:40:29  IO Worker #1 INFO import     3/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 14 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-01 10:40:59  IO Worker #3 INFO import     3/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 14 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
```

**Comment by joshua-mir** [comment]
You'll probably need to wait until you have more peers before seeing any significant progress. An archive sync can take a few weeks, so let me know if you still haven't moved by tomorrow.

**Comment by chakhalyan1** [comment]
OK. thanks

**Comment by chakhalyan1** [comment]
Hi joshua-mir.
from yesterday my not continue sync process. printed following logs.
`2019-04-02 09:13:43  IO Worker #0 INFO import  Syncing #2714664 0xd0f0…6a66     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    0 Qed    #16448    4/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 53 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-02 09:13:48  IO Worker #2 INFO import  Syncing #2714664 0xd0f0…6a66     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    0 Qed    #18924    4/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 53 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
2019-04-02 09:13:53  IO Worker #1 INFO import  Syncing #2714664 0xd0f0…6a66     0.00 blk/s    0.0 tx/s    0.0 Mgas/s      0+    0 Qed    #21353    4/ 6 peers   80 KiB chain 0 bytes db 0 bytes queue 53 MiB sync  RPC:  0 conn,    0 req/s,    0 µs
`

**Comment by joshua-mir** [comment]
Can you please try changing `sync=debug` to `sync=trace`?

**Comment by chakhalyan1** [comment]
have done will wit if any changes let You know

**Comment by cxd1995** [comment]
We also encounter similar error when syncing the block number 7280086.
![image](https://user-images.githubusercontent.com/45622921/55447878-31f68280-55f8-11e9-8f32-29a87397d099.png)
![image](https://user-images.githubusercontent.com/45622921/55447882-3458dc80-55f8-11e9-9bc2-67b49b8c28f3.png)
Guess it may be caused the update of Ethereum client. And we empty the data and synchronize from the start. BUT the error is not solved.

**Comment by joshua-mir** [comment]
@cxd1995 - the issue you are seeing is that your parity doesn't support recent hard forks so it will not sync past the fork blocks. Please update your node to the latest stable version.

**Comment by chakhalyan1** [comment]
Hi All, I have one question can help anyone ?
I what listen rejected(means that a transaction was not added to the queue) transaction is possible or not? 
Parity Ethereum version: 2.5.0
Operating system: Linux

**Comment by joshua-mir** [comment]
@chakhalyan1 you might be able to using `-l txqueue` logging.

**Comment by chakhalyan1** [comment]
that queue included  rejected transactions ?

**Comment by joshua-mir** [comment]
@chakhalyan1 the txqueue logging module logs when transactions are rejected.

**Comment by chakhalyan1** [comment]
> @chakhalyan1 the txqueue logging module logs when transactions are rejected.

ok, thanks.
will testing then  let You know

**Comment by chakhalyan1** [comment]
for me always problem depended to parity, can You help me what is the right way setup ? 
i have stop and start parity sync process went down  202687 when finished sync

**Comment by joshua-mir** [comment]
@chakhalyan1 there are various other open issues about stalling sync - that question would probably be better in one of those, and if you can't find one that seems appropriate, feel free to open a new issue.

**Comment by chakhalyan1** [comment]
thanks

**Comment by chakhalyan1** [comment]
> @chakhalyan1 you might be able to using `-l txqueue` logging.

Hi @joshua-mir.
I have  started my parity node `parity -l txqueue` it`s helped me. i have getting now rejected transaction on the parity logs. but i am using web3 and subscribed to pending transactions  but i thing i did not get rejected transactions. can You help how can listen it?

**Comment by joshua-mir** [comment]
There isn't an rpc method accessible via web3 or otherwise that gives you rejected transactions..

**Comment by chakhalyan1** [comment]
> There isn't an rpc method accessibl

if subscribed with pending transactions its also is not listing  rejected transactions?

**Comment by joshua-mir** [comment]
If they are rejected, that happens before they get into the pending transaction set

**Comment by chakhalyan1** [comment]
> If they are rejected, that happens before they get into the pending transaction set

oy ok, thanks. then i thing i can get it from parity log file. if i tack hash from log then can i query to get transaction info via web3?
